scripts/build_ios.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+PROJECT_DIR=$(pwd)
+
+# Check for the device type argument
+if [[ "$1" == "simulator" ]]; then
+    DEVICE_TYPE="simulator"
+    ARCHITECTURE="aarch64-apple-ios-sim"
+elif [[ "$1" == "device" ]]; then
+    DEVICE_TYPE="device"
+    ARCHITECTURE="aarch64-apple-ios"
+else
+    echo -e "${RED}Error: Please specify either 'simulator' or 'device' as the first argument.${DEFAULT}"
```

Comment:
Where is RED from here? Think this script is missing colors prelude from other script maybe?